bedrock/contentful/tests/test_contentful_api.py

```diff
@@ -1197,44 +1197,116 @@ def test_ContentfulPage__get_info_data(
     )
 
 
-# FURTHER TESTS TO COME
-# def test_ContentfulPage__get_content():
-#     assert False, "WRITE ME"
+@patch("bedrock.contentful.api._get_image_url")
+def test_ContentfulPage__get_split_data(mock__get_image_url, basic_contentful_page):
+    # mock self and entry data
+    basic_contentful_page.page = Mock()
```

Comment:
Also, is this the correct way to use the basic_contentful_page fixture?

Reply:
Looks like this is similar to how @stevejalim used the fixture [here](https://github.com/mozilla/bedrock/blob/9a6e65d1ff7b5d24aff12861710996f45d92cffb/bedrock/contentful/tests/test_contentful_api.py#L978)